[Upgrade Assistant] Batch reindex docs

yuliacech opened the pull request:
### Summary 
This PR adds a link to Kibana batch reindex API to the ES deprecations page. This could be helpful for users that have a lot of indices that need to be reindexed and clicking each one in the UI might be tiring for them. They could use "batch reindex" API via Dev Tools console.
The Kibana batch reindexing page now links to Kibana reindex that also links out to ES reindex page. 
I also added a list of steps that happen during the Kibana reindex task and renamed "reindexing operation" on one page to "reindexing task" for consistency with other pages. Also the "experimental" callout is now the same on all UA docs pages.

### Screenshot
<img width="1080" alt="Screenshot 2021-09-27 at 12 42 40" src="https://user-images.githubusercontent.com/6585477/134894846-fcf8f489-86c5-459d-924c-a7b346cd4323.png">


### Checklist

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [x] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials

elasticmachine commented:
Pinging @elastic/kibana-stack-management (Team:Stack Management)

yuliacech commented on docs/api/upgrade-assistant/batch_reindexing.asciidoc:
```diff
@@ -6,7 +6,7 @@
 experimental["The underlying Upgrade Assistant concepts are stable, but the APIs for managing Upgrade Assistant are experimental."]
 
-Start or resume multiple reindexing tasks in one request. Additionally, reindexing tasks started or resumed
+Start or resume multiple <<start-resume-reindex, reindexing>> tasks in one request. Additionally, reindexing tasks started or resumed
```
This should like to the Kibana [Start or resume reindex API](https://www.elastic.co/guide/en/kibana/master/start-resume-reindex.html) page

yuliacech commented on docs/api/upgrade-assistant/batch_reindexing.asciidoc:
```diff
@@ -76,7 +76,7 @@ Similar to the <<start-resume-reindex, start or resume endpoint>>, the API retur
 --------------------------------------------------
 
-<1> A list of reindex operations created, the order in the array indicates the order in which tasks will be executed.
+<1> A list of reindex tasks created, the order in the array indicates the order in which tasks will be executed.
```
Changing wording for consistency, as we use "task" everywhere else.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/)
* Commit: a33d4086b22eafd3561bbe19bf36fde3640e1ee6
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/execution/node/425/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/","number":"155293","commit":"a33d4086b22eafd3561bbe19bf36fde3640e1ee6"}]}
PIPELINE-->

yuliacech commented on docs/api/upgrade-assistant/reindexing.asciidoc:
```diff
@@ -6,7 +6,16 @@
+. Creating a new index
+. {ref}/docs-reindex.html[Reindexing] documents into the new index
+. Creating an index alias for the new index
+. Deleting the old index
```
This list doesn't contain 2 steps: stopping and starting "index group services, only applies to some indices" which happens for ml and watcher indices. This code will be removed from UA this release because system indices will have their own separate deprecation resolution process.

yuliacech commented on docs/api/upgrade-assistant/reindexing.asciidoc:
```diff
@@ -40,6 +49,6 @@ The API returns the following:
-<4> The task ID of the reindex task in {es}. Appears when the reindexing starts.
-<5> The progress of the reindexing task in {es}. Appears in decimal form, from 0 to 1.
+<4> The task ID of the {ref}/docs-reindex.html[reindex] task in {es}. Appears when the reindexing starts.
+<5> The progress of the {ref}/docs-reindex.html[reindexing] task in {es}. Appears in decimal form, from 0 to 1.
```
This should link out to ES [Reindex API](https://www.elastic.co/guide/en/elasticsearch/reference/master/docs-reindex.html) page

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/)
* Commit: 6e3201796fe8a414f559428df4bea523ebebceb1
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Execute oss-accessibility](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/execution/node/597/log)
* [Execute oss-accessibility](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/execution/node/726/log)

### History
* :broken_heart: [Build #155293](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/) failed a33d4086b22eafd3561bbe19bf36fde3640e1ee6

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/","number":"155293","commit":"a33d4086b22eafd3561bbe19bf36fde3640e1ee6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/","number":"155299","commit":"6e3201796fe8a414f559428df4bea523ebebceb1"}]}
PIPELINE-->

jrodewig commented:
@elasticmachine run elasticsearch-ci/docs

jrodewig commented on x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx:
```diff
@@ -42,16 +43,30 @@ const i18nTexts = {
   pageDescription: i18n.translate('xpack.upgradeAssistant.esDeprecations.pageDescription', {
     defaultMessage:
-      'You must resolve all critical issues before upgrading. Back up recommended. Make sure you have a current snapshot before modifying your configuration or reindexing.',
+      'You must resolve all critical issues before upgrading. Back up recommended. Make sure you have a current snapshot before modifying your configuration or reindexing. ',
```
Unless I'm missing something, making a backup and taking a snapshot are the same thing. I think we can also have a stronger lead-in for reindexing.
```suggestion
      'Resolve all critical issues before upgrading. Before making changes, ensure you have a current snapshot of your cluster. Some issues may require reindexing to resolve. ',
```

jrodewig commented on x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx:
```diff
@@ -42,16 +43,30 @@ const i18nTexts = {
+  return (
+    <EuiLink href={docLinks.links.upgradeAssistant.batchReindex} target="_blank" external={true}>
+      {i18n.translate('xpack.upgradeAssistant.esDeprecations.batchReindexingDocsLink', {
+        defaultMessage: 'Learn more about Kibana batch reindexing API.',
```
IMO It's not really clear why a user would want to use the batch reindexing API here. I updated the copy to help. However, we probably want to move some of this outside of the link text.
```suggestion
        defaultMessage: 'To start multiple reindexing tasks in a single request, use the Kibana batch reindexing API.',
```

cjcenizal replied:
Thanks for this suggestion, James! I had the same thought.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/)
* Commit: 8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Quick Commit Checks](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/execution/node/421/log)

### History
* :broken_heart: [Build #155299](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/) failed 6e3201796fe8a414f559428df4bea523ebebceb1
* :broken_heart: [Build #155293](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/) failed a33d4086b22eafd3561bbe19bf36fde3640e1ee6

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/","number":"155293","commit":"a33d4086b22eafd3561bbe19bf36fde3640e1ee6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/","number":"155299","commit":"6e3201796fe8a414f559428df4bea523ebebceb1"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/","number":"155386","commit":"8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155374/)
* Commit: 9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-112960/9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b)
* [Documentation Changes](https://kibana_112960.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/f08be5cf6be3784506c670bc43f83fb7ce32f10c) | [after](https://github.com/elastic/kibana/commit/9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 212.3KB | 212.9KB | +548.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/f08be5cf6be3784506c670bc43f83fb7ce32f10c) | [after](https://github.com/elastic/kibana/commit/9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b) | diff |
| --- | --- | --- | --- |
| `core` | 408.2KB | 408.2KB | +70.0B |

### History
* :broken_heart: [Build #155386](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/) failed 8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e
* :broken_heart: [Build #155299](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/) failed 6e3201796fe8a414f559428df4bea523ebebceb1
* :broken_heart: [Build #155293](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/) failed a33d4086b22eafd3561bbe19bf36fde3640e1ee6

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/","number":"155293","commit":"a33d4086b22eafd3561bbe19bf36fde3640e1ee6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/","number":"155299","commit":"6e3201796fe8a414f559428df4bea523ebebceb1"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/","number":"155386","commit":"8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155374/","number":"155374","commit":"9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b"}]}
PIPELINE-->

sabarasaba reviewed (approved):
Thanks for making these changes @yuliacech! code lgtm, didn't test locally

yuliacech commented:
@elasticmachine merge upstream

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155670/)
* Commit: 890d25ddcfd632b56f5eab4ce1a2646f95f72dd0
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-112960/890d25ddcfd632b56f5eab4ce1a2646f95f72dd0)
* This comment will update when the build is complete

### Failed CI Steps
* [Lint: eslint](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155670/execution/node/419/log)

### History
* :broken_heart: [Build #155386](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/) failed 8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e
* :green_heart: [Build #155374](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155374/) succeeded 9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b
* :broken_heart: [Build #155299](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/) failed 6e3201796fe8a414f559428df4bea523ebebceb1
* :broken_heart: [Build #155293](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/) failed a33d4086b22eafd3561bbe19bf36fde3640e1ee6

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/","number":"155293","commit":"a33d4086b22eafd3561bbe19bf36fde3640e1ee6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/","number":"155299","commit":"6e3201796fe8a414f559428df4bea523ebebceb1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155374/","number":"155374","commit":"9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/","number":"155386","commit":"8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155670/","number":"155670","commit":"890d25ddcfd632b56f5eab4ce1a2646f95f72dd0"}]}
PIPELINE-->

yuliacech commented:
Thank you for the review, @sabarasaba!

yuliacech commented:
Thanks a lot for your review and the suggestions, @jrodewig! 
I updated the ES deprecations page wording as you suggested and fixed the "experimental" tag on the docs pages. Could you please have another look?

kibanamachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155698/)
* Commit: ecdcdb8125a9951446f21a7550ac4fa378469f7e
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-112960/ecdcdb8125a9951446f21a7550ac4fa378469f7e)
* [Documentation Changes](https://kibana_112960.docs-preview.app.elstc.co/diff)
* Flaky suites:
  * xpack-kibana-ciGroup8

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155698/testReport)

<details><summary>Kibana Pipeline / general / Performance Tests.x-pack/test/performance/tests/reporting_dashboard·ts.performance reporting dashbaord downloaded PDF has OK status</summary>

[Link to Jenkins](https://kibana-ci.elastic.co//job/elastic+kibana+pipeline-pull-request/155698/testReport/junit/Performance%20Tests/x-pack_test_performance_tests_reporting_dashboard·ts/Kibana_Pipeline___general___performance_reporting_dashbaord_downloaded_PDF_has_OK_status)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 3 times on tracked branches: https://github.com/elastic/kibana/issues/110470

[00:00:00]                     │
[00:00:00]                       └-: performance
[00:00:00]                         └-> "before all" hook in "performance"
[00:00:00]                         └-: reporting dashbaord
[00:00:00]                           └-> "before all" hook for "downloaded PDF has OK status"
[00:00:00]                           └-> "before all" hook for "downloaded PDF has OK status"
[00:00:00]                             │ debg resolved import for x-pack/test/performance/kbn_archives/reporting_dashboard to /dev/shm/workspace/parallel/11/kibana/x-pack/test/performance/kbn_archives/reporting_dashboard.json
[00:00:00]                             │ info importing 4 saved objects { space: undefined }
[00:00:00]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:00]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:00]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:00]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:00]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:01]                             │ succ import success
[00:00:01]                             │ info [x-pack/test/performance/es_archives/reporting_dashboard] Loading "mappings.json"
[00:00:01]                             │ info [x-pack/test/performance/es_archives/reporting_dashboard] Loading "data.json.gz"
[00:00:01]                             │ info [o.e.c.m.MetadataCreateIndexService] [node-01] [foo] creating index, cause [api], templates [], shards [1]/[1]
[00:00:01]                             │ info [x-pack/test/performance/es_archives/reporting_dashboard] Created index "foo"
[00:00:01]                             │ debg [x-pack/test/performance/es_archives/reporting_dashboard] "foo" settings {"index":{"number_of_replicas":"1","number_of_shards":"1"}}
[00:00:03]                             │ info [x-pack/test/performance/es_archives/reporting_dashboard] Indexed 10000 docs into "foo"
[00:00:03]                           └-> downloaded PDF has OK status
[00:00:03]                             └-> "before each" hook: global before each for "downloaded PDF has OK status"
[00:00:03]                             │ debg navigating to dashboards url: http://localhost:61111/app/dashboards
[00:00:03]                             │ debg navigate to: http://localhost:61111/app/dashboards
[00:00:03]                             │ debg browser[INFO] http://localhost:61111/login?next=%2Fapp%2Fdashboards%3F_t%3D1632744717033 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:03]                             │
[00:00:03]                             │ debg browser[INFO] http://localhost:61111/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:03]                             │ debg ... sleep(700) start
[00:00:04]                             │ debg ... sleep(700) end
[00:00:04]                             │ debg returned from get, calling refresh
[00:00:05]                             │ debg browser[INFO] http://localhost:61111/login?next=%2Fapp%2Fdashboards%3F_t%3D1632744717033 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:05]                             │
[00:00:05]                             │ debg browser[INFO] http://localhost:61111/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:06]                             │ debg currentUrl = http://localhost:61111/login?next=%2Fapp%2Fdashboards%3F_t%3D1632744717033
[00:00:06]                             │          appUrl = http://localhost:61111/app/dashboards
[00:00:06]                             │ debg TestSubjects.find(kibanaChrome)
[00:00:06]                             │ debg Find.findByCssSelector('[data-test-subj="kibanaChrome"]') with timeout=60000
[00:00:06]                             │ debg Found login page
[00:00:06]                             │ debg TestSubjects.setValue(loginUsername, test_user)
[00:00:06]                             │ debg TestSubjects.click(loginUsername)
[00:00:06]                             │ debg Find.clickByCssSelector('[data-test-subj="loginUsername"]') with timeout=10000
[00:00:06]                             │ debg Find.findByCssSelector('[data-test-subj="loginUsername"]') with timeout=10000
[00:00:06]                             │ERROR browser[SEVERE] http://localhost:61111/api/licensing/info - Failed to load resource: the server responded with a status of 401 (Unauthorized)
[00:00:07]                             │ debg TestSubjects.setValue(loginPassword, changeme)
[00:00:07]                             │ debg TestSubjects.click(loginPassword)
[00:00:07]                             │ debg Find.clickByCssSelector('[data-test-subj="loginPassword"]') with timeout=10000
[00:00:07]                             │ debg Find.findByCssSelector('[data-test-subj="loginPassword"]') with timeout=10000
[00:00:07]                             │ debg TestSubjects.click(loginSubmit)
[00:00:07]                             │ debg Find.clickByCssSelector('[data-test-subj="loginSubmit"]') with timeout=10000
[00:00:07]                             │ debg Find.findByCssSelector('[data-test-subj="loginSubmit"]') with timeout=10000
[00:00:07]                             │ debg Find.waitForDeletedByCssSelector('.kibanaWelcomeLogo') with timeout=10000
[00:00:07]                             │ proc [kibana]   log   [12:12:00.942] [info][plugins][routes][security] Logging in with provider "basic" (basic)
[00:00:08]                             │ debg browser[INFO] http://localhost:61111/app/dashboards?_t=1632744717033 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:08]                             │
[00:00:08]                             │ debg browser[INFO] http://localhost:61111/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:08]                             │ debg Find.findByCssSelector('[data-test-subj="kibanaChrome"]') with timeout=60000
[00:00:09]                             │ debg Find.findByCssSelector('[data-test-subj="kibanaChrome"] nav:not(.ng-hide)') with timeout=60000
[00:00:11]                             │ debg browser[INFO] http://localhost:61111/app/dashboards?_t=1632744723597 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:11]                             │
[00:00:11]                             │ debg browser[INFO] http://localhost:61111/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:11]                             │ debg Finished login process currentUrl = http://localhost:61111/app/dashboards
[00:00:11]                             │ debg ... sleep(501) start
[00:00:12]                             │ debg ... sleep(501) end
[00:00:12]                             │ debg in navigateTo url = http://localhost:61111/app/dashboards
[00:00:12]                             │ debg Waiting up to 20000ms for dashboard landing page...
[00:00:12]                             │ debg onDashboardLandingPage
[00:00:12]                             │ debg TestSubjects.exists(dashboardLandingPage)
[00:00:12]                             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="dashboardLandingPage"]') with timeout=5000
[00:00:12]                             │ debg Load Saved Dashboard dashboard
[00:00:12]                             │ debg gotoDashboardLandingPage
[00:00:12]                             │ debg onDashboardLandingPage
[00:00:12]                             │ debg TestSubjects.exists(dashboardLandingPage)
[00:00:12]                             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="dashboardLandingPage"]') with timeout=5000
[00:00:12]                             │ debg searchForItemWithName: dashboard
[00:00:12]                             │ debg TestSubjects.find(tableListSearchBox)
[00:00:12]                             │ debg Find.findByCssSelector('[data-test-subj="tableListSearchBox"]') with timeout=10000
[00:00:13]                             │ debg isGlobalLoadingIndicatorVisible
[00:00:13]                             │ debg TestSubjects.exists(globalLoadingIndicator)
[00:00:13]                             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="globalLoadingIndicator"]') with timeout=1500
[00:00:13]                             │ debg TestSubjects.exists(globalLoadingIndicator-hidden)
[00:00:13]                             │ debg Find.existsByCssSelector('[data-test-subj="globalLoadingIndicator-hidden"]') with timeout=100000
[00:00:14]                             │ debg TestSubjects.click(dashboardListingTitleLink-dashboard)
[00:00:14]                             │ debg Find.clickByCssSelector('[data-test-subj="dashboardListingTitleLink-dashboard"]') with timeout=10000
[00:00:14]                             │ debg Find.findByCssSelector('[data-test-subj="dashboardListingTitleLink-dashboard"]') with timeout=10000
[00:00:14]                             │ debg isGlobalLoadingIndicatorVisible
[00:00:14]                             │ debg TestSubjects.exists(globalLoadingIndicator)
[00:00:14]                             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="globalLoadingIndicator"]') with timeout=1500
[00:00:14]                             │ debg browser[INFO] http://localhost:61111/app/dashboards#/view/37b49c50-2dc6-11eb-8af3-cb3aa84dbabd?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now)) 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:14]                             │
[00:00:14]                             │ debg browser[INFO] http://localhost:61111/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:16]                             │ debg --- retry.tryForTime error: [data-test-subj="globalLoadingIndicator"] is not displayed
[00:00:16]                             │ debg TestSubjects.exists(globalLoadingIndicator-hidden)
[00:00:16]                             │ debg Find.existsByCssSelector('[data-test-subj="globalLoadingIndicator-hidden"]') with timeout=100000
[00:00:16]                             │ debg TestSubjects.missingOrFail(dashboardLandingPage)
[00:00:16]                             │ debg Find.waitForDeletedByCssSelector('[data-test-subj="dashboardLandingPage"]') with timeout=10000
[00:00:17]                             │ debg openPdfReportingPanel
[00:00:17]                             │ debg openShareMenuItem title:PDF Reports
[00:00:17]                             │ debg TestSubjects.exists(shareContextMenu)
[00:00:17]                             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="shareContextMenu"]') with timeout=2500
[00:00:18]                             │ info [o.e.c.m.MetadataCreateIndexService] [node-01] [.async-search] creating index, cause [api], templates [], shards [1]/[0]
[00:00:20]                             │ debg --- retry.tryForTime error: [data-test-subj="shareContextMenu"] is not displayed
[00:00:20]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:20]                             │ debg TestSubjects.click(shareTopNavButton)
[00:00:20]                             │ debg Find.clickByCssSelector('[data-test-subj="shareTopNavButton"]') with timeout=10000
[00:00:20]                             │ debg Find.findByCssSelector('[data-test-subj="shareTopNavButton"]') with timeout=10000
[00:00:21]                             │ debg Find.findByCssSelector('div.euiContextMenuPanel') with timeout=10000
[00:00:22]                             │ debg TestSubjects.click(sharePanel-PDFReports)
[00:00:22]                             │ debg Find.clickByCssSelector('[data-test-subj="sharePanel-PDFReports"]') with timeout=10000
[00:00:22]                             │ debg Find.findByCssSelector('[data-test-subj="sharePanel-PDFReports"]') with timeout=10000
[00:00:23]                             │ debg Find.waitForElementStale with timeout=10000
[00:00:23]                             │ debg TestSubjects.click(generateReportButton)
[00:00:23]                             │ debg Find.clickByCssSelector('[data-test-subj="generateReportButton"]') with timeout=10000
[00:00:23]                             │ debg Find.findByCssSelector('[data-test-subj="generateReportButton"]') with timeout=10000
[00:00:24]                             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_7.16.0_001/Rd_IhRxcTeKAXl892s0Ezg] update_mapping [_doc]
[00:00:24]                             │ debg getReportURL
[00:00:24]                             │ debg TestSubjects.getAttribute(downloadCompletedReportButton, href, tryTimeout=120000, findTimeout=60000)
[00:00:24]                             │ debg TestSubjects.find(downloadCompletedReportButton)
[00:00:24]                             │ debg Find.findByCssSelector('[data-test-subj="downloadCompletedReportButton"]') with timeout=60000
[00:01:25]                             │ debg --- retry.tryForTime error: Waiting for element to be located By(css selector, [data-test-subj="downloadCompletedReportButton"])
[00:01:25]                             │      Wait timed out after 61211ms
[00:01:26]                             │ debg TestSubjects.find(downloadCompletedReportButton)
[00:01:26]                             │ debg Find.findByCssSelector('[data-test-subj="downloadCompletedReportButton"]') with timeout=60000
[00:02:27]                             │ debg --- retry.tryForTime error: Waiting for element to be located By(css selector, [data-test-subj="downloadCompletedReportButton"])
[00:02:27]                             │      Wait timed out after 61192ms
[00:02:27]                             │ info Taking screenshot "/dev/shm/workspace/parallel/11/kibana/x-pack/test/functional/screenshots/failure/performance reporting dashbaord downloaded PDF has OK status.png"
[00:02:28]                             │ info Current URL is: http://localhost:61111/app/dashboards#/view/37b49c50-2dc6-11eb-8af3-cb3aa84dbabd?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))
[00:02:28]                             │ info Saving page source to: /dev/shm/workspace/parallel/11/kibana/x-pack/test/performance/failure_debug/html/performance reporting dashbaord downloaded PDF has OK status.html
[00:02:28]                             └- ✖ fail: performance reporting dashbaord downloaded PDF has OK status
[00:02:28]                             │      Error: retry.tryForTime timeout: TimeoutError: Waiting for element to be located By(css selector, [data-test-subj="downloadCompletedReportButton"])
[00:02:28]                             │ Wait timed out after 61192ms
[00:02:28]                             │     at /dev/shm/workspace/parallel/11/kibana/node_modules/selenium-webdriver/lib/webdriver.js:842:17
[00:02:28]                             │     at runMicrotasks (<anonymous>)
[00:02:28]                             │     at processTicksAndRejections (internal/process/task_queues.js:95:5)
[00:02:28]                             │       at onFailure (/dev/shm/workspace/parallel/11/kibana/test/common/services/retry/retry_for_success.ts:17:9)
[00:02:28]                             │       at retryForSuccess (/dev/shm/workspace/parallel/11/kibana/test/common/services/retry/retry_for_success.ts:57:13)
[00:02:28]                             │       at RetryService.tryForTime (/dev/shm/workspace/parallel/11/kibana/test/common/services/retry/retry.ts:22:12)
[00:02:28]                             │       at TestSubjects.getAttribute (/dev/shm/workspace/parallel/11/kibana/test/functional/services/common/test_subjects.ts:190:12)
[00:02:28]                             │       at ReportingPageObject.getReportURL (test/functional/page_objects/reporting_page.ts:36:17)
[00:02:28]                             │       at Context.<anonymous> (test/performance/tests/reporting_dashboard.ts:50:7)
[00:02:28]                             │       at Object.apply (/dev/shm/workspace/parallel/11/kibana/node_modules/@kbn/test/target_node/functional_test_runner/lib/mocha/wrap_function.js:87:16)
[00:02:28]                             │ 
[00:02:28]                             │ 

```

#### Stack Trace
```
Error: retry.tryForTime timeout: TimeoutError: Waiting for element to be located By(css selector, [data-test-subj="downloadCompletedReportButton"])
Wait timed out after 61192ms
    at /dev/shm/workspace/parallel/11/kibana/node_modules/selenium-webdriver/lib/webdriver.js:842:17
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at onFailure (/dev/shm/workspace/parallel/11/kibana/test/common/services/retry/retry_for_success.ts:17:9)
    at retryForSuccess (/dev/shm/workspace/parallel/11/kibana/test/common/services/retry/retry_for_success.ts:57:13)
    at RetryService.tryForTime (/dev/shm/workspace/parallel/11/kibana/test/common/services/retry/retry.ts:22:12)
    at TestSubjects.getAttribute (/dev/shm/workspace/parallel/11/kibana/test/functional/services/common/test_subjects.ts:190:12)
    at ReportingPageObject.getReportURL (test/functional/page_objects/reporting_page.ts:36:17)
    at Context.<anonymous> (test/performance/tests/reporting_dashboard.ts:50:7)
    at Object.apply (/dev/shm/workspace/parallel/11/kibana/node_modules/@kbn/test/target_node/functional_test_runner/lib/mocha/wrap_function.js:87:16)
```
</details>

---

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/1ad85249e146c0afb14668a0c002409777273e56) | [after](https://github.com/elastic/kibana/commit/ecdcdb8125a9951446f21a7550ac4fa378469f7e) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 212.3KB | 213.1KB | +824.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/1ad85249e146c0afb14668a0c002409777273e56) | [after](https://github.com/elastic/kibana/commit/ecdcdb8125a9951446f21a7550ac4fa378469f7e) | diff |
| --- | --- | --- | --- |
| `core` | 408.2KB | 408.3KB | +70.0B |

### History
* :broken_heart: [Build #155670](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155670/) failed 890d25ddcfd632b56f5eab4ce1a2646f95f72dd0
* :broken_heart: [Build #155386](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/) failed 8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e
* :green_heart: [Build #155374](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155374/) succeeded 9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b
* :broken_heart: [Build #155299](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/) failed 6e3201796fe8a414f559428df4bea523ebebceb1
* :broken_heart: [Build #155293](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/) failed a33d4086b22eafd3561bbe19bf36fde3640e1ee6

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155293/","number":"155293","commit":"a33d4086b22eafd3561bbe19bf36fde3640e1ee6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155299/","number":"155299","commit":"6e3201796fe8a414f559428df4bea523ebebceb1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155374/","number":"155374","commit":"9071fdf2ce4b949f492fcb0b4cfc5ea2f657bd0b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155386/","number":"155386","commit":"8cb8b9c7d1c092e2a330aebdb4c37949e6a7e83e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155670/","number":"155670","commit":"890d25ddcfd632b56f5eab4ce1a2646f95f72dd0"},{"status":"UNSTABLE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155698/","number":"155698","commit":"ecdcdb8125a9951446f21a7550ac4fa378469f7e"}]}
PIPELINE-->

jrodewig reviewed (approved):
LGTM. Thanks @yuliacech!

cjcenizal reviewed (approved):
Code and screenshot looks great! Thanks for doing this, Yulia. I didn't test the links or test locally.
